test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataTest.cs

```diff
@@ -418,7 +471,7 @@ public void PropertiesProperty_CallsProvider()
 
         [Theory]
         [MemberData(nameof(PropertyNamesAndOrdersTheoryData))]
-        public void PropertiesProperty_OrdersPropertyNamesUsingOrder_ThenAlphabetically(
+        public void PropertiesProperty_OrdersPropertyNamesUsingOrder_ThenAsProvided(
```

Comment:
change is mentioned in the commit message and description. I caught these two nits (incorrect names) while updating the class.